airflow/providers/google/marketing_platform/operators/analytics.py

```diff
@@ -169,13 +173,273 @@ def __init__(
         self.account_id = account_id
         self.web_property_id = web_property_id
         self.api_version = api_version
-        self.gcp_connection_id = gcp_connection_id
+        self.gcp_conn_id = gcp_conn_id
 
     def execute(self, context):
         hook = GoogleAnalyticsHook(
-            api_version=self.api_version, gcp_connection_id=self.gcp_connection_id
+            api_version=self.api_version, gcp_conn_id=self.gcp_conn_id
         )
         result = hook.list_ad_words_links(
             account_id=self.account_id, web_property_id=self.web_property_id,
         )
         return result
+
+
+class GoogleAnalyticsDataImportUploadOperator(BaseOperator):
+    """
+    Take a file from Cloud Storage and uploads it to GA via data import API.
+
+    :param storage_bucket: The Google cloud storage bucket where the file is stored.
+    :type storage_bucket: str
+    :param storage_name_object: The name of the object in the desired Google cloud
+          storage bucket. (templated) If the destination points to an existing
+          folder, the file will be taken from the specified folder.
+    :type storage_name_object: str
+    :param account_id: The GA account Id (long) to which the data upload belongs
```

Comment:
;)
```suggestion
    :param account_id: The GA account Id (long) to which the data upload belongs.
```
And few more. ;)